src/api/utils/project.py

```
@@ -922,12 +924,15 @@ def delete_project(pk):
-            print(f"Delete Project: {err}")
-            transaction.savepoint_rollback(sid)
+    with ExitStack() as stack:
+        if instance.is_demo:
+            stack.enter_context(suppress_all_notifications())
+        with transaction.atomic():
+            sid = transaction.savepoint()
+            try:
+                delete_instance_and_related_objects(instance)
+                transaction.savepoint_commit(sid)
+                print("project deleted")
+            except Exception as err:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)